tests/ci_build/ci_build.sh

```diff
@@ -114,13 +119,14 @@ DOCKER_IMG_NAME=$(echo "${DOCKER_IMG_NAME}" | tr '[:upper:]' '[:lower:]')
 # skip with_the_same_user for non-linux
 uname=`uname`
 if [[ "$uname" == "Linux" ]]; then
-    PRE_COMMAND="tests/ci_build/with_the_same_user"
+    PRE_COMMAND="/workspace/tests/ci_build/with_the_same_user"
```

Comment:
Why not use the $workspace variable?

Reply:
WORKSPACE is a variable pointing to a host directory, this command is run inside docker. Workdir we don't want to use because it's variable, user defined. Below in the -v arg, the second param is hardcoded to '/workspace'. That's the folder we want to target.

Reply:
Maybe add a comment? I was also wondering... it's not clear.

Reply:
Updated.